src/firefly/js/tables/ui/TablePanelOptions.jsx

```diff
@@ -244,6 +246,20 @@ export const ColumnOptions = React.memo(({tbl_id, tbl_ui_id, ctm_tbl_id, onChang
     );
 });
 
+function TablePref({tbl_id, tbl_ui_id}) {
+
+    const prefApplied = useStoreConnector(() => getTableUiById(tbl_ui_id)?.[PREF_APPLIED]);
+    if (!prefApplied) return null;
+
+    return (
+        <div className='TablePanelOptions__pref'>
+            Preferences were applied to this table.
+            Click <div className='ff-href' style={{margin: '0 4px'}} onClick={() => resetTblOptions(tbl_id, tbl_ui_id)}> here </div> to clear it.
```

Comment:
I thought we were getting away from links to act as buttons and only using them to open other pages.  You probably should use a button here. 

I see you are trying to make it low key, which is a good idea. I sort of like the link better but I think we made a decision a couple of years ago to use buttons.

Reply:
I think a button between words or in a sentence is not very clean.  But, I will change to whatever is decided at the end of the review.
Another option is to not have the 'clear' function at all.  Just use 'Reset'.  It will clear filters and pagesize as well, but it will most likely not have any if it's the first thing you do after seeing the table.

Reply:
The button would have to be something link: 
```js
Preferences were applied to this table: <button> Remove</button>
```

Reply:
I don't done know about just using reset.